Answers.md

```diff
@@ -1,3 +1,15 @@
 ### Your responses to the short answer questions should be laid out here using markdown.
 
 For help with markdown syntax, [go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+## 1. Containers are faster and use less resources than virtual machines.
+
+## 2. It is the server port that the App is running on
+
+## 3. Kubernetes simplifies the process to deploy applications
+
+## 4. I have no idea...
```

Comment:
To scale a Kubernetes cluster, you need to edit the config files and update the `replicas` value. Once you've done that, run the `apply` command from kubernetes cli with the updated file.